libethcore/KeyManager.cpp

```diff
@@ -342,6 +342,19 @@ string const& KeyManager::accountName(Address const& _address) const
 	}
 }
 
+void KeyManager::changeName(Address const& _address, std::string const& _name)
+{
+	try
+	{
+		m_keyInfo.at(_address).accountName = _name;
+	}
+	catch (...)
+	{
+		return;
```

Comment:
I fixed it in latest pr.
Other methods in this class also do not throw on invalid input.